docs/src/content/docs/commands/antiraid/index.md

```
@@ -0,0 +1,44 @@
+---
+title: Antiraid Commands
+description: Complete guide to Antiraid module commands and features
+---
+
+# 📦 Antiraid Commands
+
+Protect your group from spam-join attacks with AntiRaid.
+### Admin Commands:
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)